core/src/epicli/data/common/defaults/configuration/postgres-exporter.yml

```diff
@@ -0,0 +1,19 @@
+kind: configuration/postgres-exporter
+title: Postgres exporter
+name: default
+specification:
+  version: 0.8.0
+  file_name: postgres_exporter_v0.8.0_linux-amd64.tar.gz
+  description: Service that runs Postgres Exporter
```

Comment:
The same, I would specify this variable in role's defaults.

Reply:
Moved to defaults